exporter/tanzuobservabilityexporter/metrics_test.go

```diff
@@ -761,6 +761,190 @@ func TestDeltaHistogramDataPointConsumerMissingBuckets(t *testing.T) {
 	assert.Equal(t, int64(1), report.Malformed())
 }
 
+func TestSummaries(t *testing.T) {
+	summaryMetric := newMetric("test.summary", pdata.MetricDataTypeSummary)
+	summary := summaryMetric.Summary()
+	dataPoints := summary.DataPoints()
+	dataPoints.EnsureCapacity(2)
+
+	dataPoint := dataPoints.AppendEmpty()
+	setQuantileValues(dataPoint, 0.1, 100.0, 0.5, 200.0, 0.9, 300.0, 0.99, 400.0)
+	setTags(map[string]interface{}{"foo": "bar"}, dataPoint.Attributes())
+	dataPoint.SetCount(10)
+	dataPoint.SetSum(5000.0)
+	setDataPointTimestamp(1645123456, dataPoint)
+
+	dataPoint = dataPoints.AppendEmpty()
+	setQuantileValues(dataPoint, 0.2, 75.0, 0.5, 125.0, 0.8, 175.0, 0.95, 225.0)
+	setTags(map[string]interface{}{"bar": "baz"}, dataPoint.Attributes())
+	dataPoint.SetCount(15)
+	dataPoint.SetSum(3000.0)
+	setDataPointTimestamp(1645123556, dataPoint)
+
+	sender := &mockGaugeSender{}
+	consumer := newSummaryConsumer(sender, componenttest.NewNopTelemetrySettings())
+
+	assert.Equal(t, pdata.MetricDataTypeSummary, consumer.Type())
+
+	var errs []error
+	consumer.Consume(summaryMetric, &errs)
+
+	// Summaries don't have internal metrics, but this is here for 100% test coverage.
```

Comment:
If there's no requirement for 100% line coverage, I'd rather avoid this. I don't think it adds much value

Reply:
Looks like the requirement is (soon to be) 90%: 
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.codecov.yml#L16

Reply:
Done. Line removed.